lib/external_interaction/rubix/rubix_external_rest_api.dart

```diff
@@ -25,4 +28,26 @@ Future<Response> handleTransactionRequest(Request request) async {
   return Response.ok('Transaction request received');
 }
 
+Future<bool> externalApi({required String did,required String peerId,required String orgName,required String callBackUrl,required Token orgAccessToken })async {
```

Comment:
at the last, this function shall move to another file as it is generic. Shall create a new module, - sky_outgoing_calls